consensus/istanbul/core/roundchange.go

```diff
@@ -29,51 +29,24 @@ import (
 
 // sendRoundChange broadcasts a ROUND CHANGE message with the current desired round.
 func (c *core) sendRoundChange() {
-	logger := c.newLogger("func", "sendRoundChange")
-
-	msg, err := c.buildRoundChangeMsg(c.current.DesiredRound())
-	if err != nil {
-		logger.Error("Could not build round change message", "err", msg)
```

Comment:
I'm picking this as a random example, but I think my preference is to potentially hang and be unable to progress rather than panicking.